__tests__/app.test.js

```diff
@@ -156,4 +156,55 @@ describe("/api/articles/:article_id/comments", () => {
         expect(response.body.comments).toStrictEqual([]);
       });
   });
-})
\ No newline at end of file
+})
+describe("POST /api/articles/:article_id/comments",() =>{
+  test('201: responds with a new comment',() => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This article is lit!",
+    };
+    return request(app)
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then((response) =>{
+      expect(response.body.comment.author).toBe("butter_bridge");
+      expect(response.body.comment.body).toBe("This article is lit!");
+      expect(response.body.comment).toMatchObject({
+        comment_id: expect.any(Number),
+        body: expect.any(String),
+        article_id: expect.any(Number),
+        author: expect.any(String),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+      });  
+
+    })
+  })
+  test("POST:404 sends an appropriate status and error message when article id given is valid but non-existen id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This article is lit!",
+    };
+    return request(app)
+      .post("/api/articles/99/comments")
+      .send(newComment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("this article_id doesn't exist");
+      });
+  });
+  test("POST:400 responds with a message when given an invalid id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This article is lit!",
+    };
+    return request(app)
+      .post("/api/articles/bad_id/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
```

Comment:
There are couple more tests needed here - 
Status 400 - missing required fields for example `body : "test"`
Status 404 - username doesn't exist in the database
Status 201 - ignores unnecessary fields